### PR TITLE
added rating and modal

### PR DESCRIPTION
**Pull request by haarsh24**

### CODE REVIEW

**Review by xii-vi — APPROVED** [review]
_Looks go to merge_

**Review by yashpatel1797 — APPROVED** [review]
good to merge

File: components/components-style.css [hunk 1]
```diff
@@ -95,7 +97,7 @@
   flex-direction: row;
   margin: 0.5rem;
   align-items: center;
- 
+
   height: 2rem;
   font-size: 1.4rem;
   /* scroll-behavior: auto; */
```
Comment (horsemaker) [inline components/components-style.css:103]: You could remove this comment if not needed.

File: components/components-style.css [hunk 2]
```diff
@@ -120,11 +122,11 @@ a {
   color: var(--font-color);
 }
 
-iframe {
+/* iframe {
   max-inline-size: 100%;
   block-size: auto;
   overflow-x: scroll;
-}
+} */
```
Comment (horsemaker) [inline components/components-style.css:129]: You could remove this comment if not needed.

File: components/components-style.css (hunk 1 above)
Comment (pradipgharge) [inline components/components-style.css:103]: Do not push commented code

File: components/rating.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  .star-rating {
+    direction: rtl;
+    display: inline-block;
+    cursor: default;
+  }
+  .star-rating .star-input[type="radio"] {
+    display: none;
+  }
+  .star-rating .star-label {
+    color: #bbb;
```
Comment (pradipgharge) [inline components/rating.css:18]: use color variable here

File: components/rating.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    font-size: 18px;
```
Comment (pradipgharge) [inline components/rating.css:19]: you can use rems here

**Review by horsemaker — APPROVED** [review]
Good module code! 👌
Liked you UI! 👍

**Review by pradipgharge — APPROVED** [review]
Suggested some changes, rest looks good to merge

**Pull request merged**